docs/tutorial-3.md

```diff
@@ -0,0 +1,201 @@
+  // and distinguishes from a normal Fact by having a run_id
+  // It will also be stored in the Fact table in Cicero db
+  // after a Run has completed
+  output = { start }:
+    let cfg = start.value.${name}.start; in
+    {
+      success.${name} = {
+        ok = true;
+        revision = cfg.sha;
+      } // lib.optionalAttrs (cfg ? ref) {
+        inherit (cfg) ref default_branch;
+      };
+    };
+
+  // The job describes, what is executed
+  // when the Nomad task is created and run
+  job = { start }:
+    let cfg = start.value.${name}.start; in
+
+    // A chain allows to chain multiple functions
+    // into one array.
+    // The functions in the array a called consecutively
+    std.chain args [
+
+      // simpleJob just escapes the provided name
+      // stores a set of {${name}.group.${name}.task.${name} = task;};
+      // and calls the next job
+      actionLib.simpleJob
+
+      // reportStatus tries to report to the github status api
+      // if a statuses_url is available
+      (std.github.reportStatus cfg.statuses_url or null)
+
+      // the template set creates a netrc file
+      // inside the Nomad Task
+      {
+        template = std.data-merge.append [{
+          destination = "secrets/netrc";
+          data = ''
+            machine github.com
+            login git
+            password {{with secret "kv/data/cicero/github"}}{{.Data.data.token}}{{end}}
+          '';
+        }];
+      }
+
+      // git.clone actually starts to clone
+      // the url provided by the input Fact
+      (std.git.clone cfg)
+
+      // the resources set is used
+      // for configuring the Nomad Task
+      {
+        resources = {
+          cpu = 3000;
+          memory = 1024 * 3;
+        };
+      }
+
+      // nix.build actually runs the nix build of the flake.nix
+      std.nix.build
+
+
+      // another template set, which is used for storing
+      // docker credentials in the Nomad Task
+      {
+        env.REGISTRY_AUTH_FILE = "/secrets/auth.json";
+
+        template = std.data-merge.append [{
+          destination = "secrets/auth.json";
+          data = ''{
+            "auths": {
+              "docker.infra.aws.iohkdev.io": {
+                "auth": "{{with secret "kv/data/cicero/docker"}}{{with .Data.data}}{{base64Encode (print .user ":" .password)}}{{end}}{{end}}"
+              }
+            }
+          }'';
+        }];
+      }
+
+      // this bash script will actually execute
+      // a specific flake output, which
+      // creates and pushes a docker image to a registry
+      (std.script "bash" ''
+        set -x
+        nix run .#laresImage.copyToRegistry
+      '')
+    ];
+
+```
+
+## Create Action in WebUI
+
+Click on "List Actions"
+
+![Cicero WebUI List lares](./cicero_webui_list_actions_lares.png "Cicero WebUI List lares")
+
+Click on "lares/ci"
+
+![Cicero WebUI Create lares](./cicero_webui_create_action_lares.png "Cicero WebUI Create lares")
+
```

Comment:
![image](https://user-images.githubusercontent.com/10901543/166006114-2916ee4b-6e02-41d5-8124-195e50e26bd5.png)

lares CI build was changed, currently there are 2 actions available. So one cannot use lares repository as example. In particular next step is not possible to achieve, there seems to be different.

If you choose inputs / latests you will see json:
```json
{
	"lares/ci": start: {
		clone_url:       string
		sha:             string
		statuses_url?:   string
		ref?:            "refs/heads/\(default_branch)"
		default_branch?: string
	}
}
```
e.g. how to fill `stateless_url` ?

CI of existing projects might change. Perhaps it would be possible to create simple projects (e.g. for Rust, Haskell, Scala as UI is using) and use them as examples? It seems like a lot of work.

Reply:
I will create a new repo which contains examples and will update the handbook afterwards: https://github.com/input-output-hk/cicero-example

Reply:
https://cicero-handbook.infra.aws.iohkdev.io/tutorial-3.html has been updated to now use the [rust exampe](https://github.com/input-output-hk/cicero-example) from cicero-example repository.